etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json

```diff
@@ -1615,11 +1615,11 @@
                 "datasource": "",
                 "hide": 0,
                 "includeAll": false,
-                "label": "Cluster Id",
+                "label": "Cluster Name",
                 "multi": false,
                 "name": "cluster_id",
```

Comment:
Why is the variable for `label: Cluster Name` called `cluster_id`?

Reply:
@shirshendu Cluster Name is actually used for showing on grafana  dashboard but the cluster id is the name of variable which can be name or ID both.

Reply:
Earlier this `cluster_id` variable was used to dig into the `tendrl.clusters.*` namespace, so `cluster_id` made sense. But now the situation is that this variable is used to dig into the `tendrl.names.*` namespace. So It would make sense to call this `cluster_name` as well.

In my opinion, it is only an implementation detail that the name is identical to the cluster ID if the user did not supply a name. Thoughts?

Reply:
@shirshendu If a user didn't supply the name then the name will itself be cluster_id only so that's why I left this variable as is.

Reply:
I don't see the confusion here though; as you say, the *name* will be set as `asdf-gakjsd-gkjasdkajsd-kasjdasd` instead of `cluster_1`.
In the end, `asdf-gakjsd-gkjasdkajsd-kasjdasd` is set as the name if the user didn't supply one.
For the purposes of monitoring, we are no longer bothered with the cluster ID. The data extracted from grafana is from the names folder and not from the clusters folder. The fact that it is symlinked back to `clusters/cluster_id` folder is invisible to this code.
Or perhaps I'm misunderstanding something. Can we get on a call about this?

Reply:
@shirshendu As it's just a readability issue can we create another Issue and fix it later after the release?

The impacts due to the cluster_id to cluster_name variable change are on different components.
1. Changes in all the query in grafana and also the links between the different dashboards need this change
2. Alerting logic needs to be changed according to the variable change.
3. UI Links to grafana dashboard will be broken so the change in the UI will be required.

Once I made this change it will require a good amount of testing on grafana and alerting side. Thoughts? @shirshendu @nthomas-redhat @shtripat

Reply:
@cloudbehl Ack, putting this on hold.

Reply:
@shirshendu created an Issue https://github.com/Tendrl/monitoring-integration/issues/412 for the above change required. Thanks